lib/yarv.rb

```diff
@@ -298,6 +298,8 @@ def initialize(selfo, iseq)
           @insns << PutSelf.new(selfo)
         in :putstring, string
           @insns << PutString.new(string)
+        in :send, { mid:, orig_argc:, flag: }, block_iseq
+          @insns << Send.new(CallData.new(mid, orig_argc, flag), InstructionSequence.new(selfo, block_iseq))
```

Comment:
@kddnewton wondering if you have advice here, even when I remove these changes, stree seems to break when trying to check, format or debug this file:
```shell
> stree debug lib/yarv.rb
[warn] lib/yarv.rb
undefined method `location' for nil:NilClass

        location: parts[0].location.to(parts[-1].location)
                          ^^^^^^^^^
/Users/.../.gem/ruby/3.1.2/gems/syntax_tree-2.0.1/lib/syntax_tree/parser.rb:1437:in `on_hshptn'
/Users/.../.gem/ruby/3.1.2/gems/syntax_tree-2.0.1/lib/syntax_tree.rb:42:in `parse'
/Users/.../.gem/ruby/3.1.2/gems/syntax_tree-2.0.1/lib/syntax_tree.rb:42:in `parse'
/Users/.../.gem/ruby/3.1.2/gems/syntax_tree-2.0.1/lib/syntax_tree.rb:49:in `format'
/Users/.../.gem/ruby/3.1.2/gems/syntax_tree-2.0.1/lib/syntax_tree/cli.rb:84:in `run'
/Users/.../.gem/ruby/3.1.2/gems/syntax_tree-2.0.1/lib/syntax_tree/cli.rb:241:in `block (2 levels) in run'
/Users/.../.gem/ruby/3.1.2/gems/syntax_tree-2.0.1/lib/syntax_tree/cli.rb:234:in `each'
/Users/.../.gem/ruby/3.1.2/gems/syntax_tree-2.0.1/lib/syntax_tree/cli.rb:234:in `block in run'
/Users/.../.gem/ruby/3.1.2/gems/syntax_tree-2.0.1/lib/syntax_tree/cli.rb:233:in `each'
/Users/.../.gem/ruby/3.1.2/gems/syntax_tree-2.0.1/lib/syntax_tree/cli.rb:233:in `run'
/Users/.../.gem/ruby/3.1.2/gems/syntax_tree-2.0.1/exe/stree:9:in `<top (required)>'
/Users/.../.gem/ruby/3.1.2/bin/stree:25:in `load'
/Users/.../.gem/ruby/3.1.2/bin/stree:25:in `<main>'
The listed files could not be formatted idempotently.
```

Reply:
Yeah, you should rebase off `main`, it was using an old version of syntax tree. It's fixed now on `main`.